packages/api/src/v2/resourcePlugin.ts

```diff
@@ -3,11 +3,13 @@
 
 import { Result } from "neverthrow";
 import { FxError, QTreeNode, TokenProvider, Void, Func, Json, Inputs } from "../index";
+import { EnvConfig } from "../schemas";
```

Comment:
As we discussed, the schema of env config should be included in solution and shouldn't be exposed to API?